src/Adapter/Cart/CartPresenter.php

```diff
@@ -212,16 +207,16 @@ public function present(Cart $cart)
 
         $total_excluding_tax = $cart->getOrderTotal(false);
         $total_including_tax = $cart->getOrderTotal(true);
+        $total_discount      = $cart->getOrderTotal(true, Cart::ONLY_DISCOUNTS);
+
+        $subtotals['tax'] = [
+            'type' => 'tax',
+            'label' => $this->translator->trans('Tax', [], 'Cart'),
+            'amount' => $this->pricePresenter->convertAndFormat(
+                $total_including_tax - $total_excluding_tax
+            ),
+        ];
 
-        if ($this->shouldShowTaxLine()) {
```

Comment:
This is what we want to forbid, this is not up to the template to decide if we display or not the line. This is the controller which need to send the right information.
The template loops on `subtotals` variable, so it must contain only the necessary line to be displayed.